lib/membrane_rtc_engine/endpoints/webrtc_endpoint.ex

```diff
@@ -107,24 +111,40 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC do
       log_metadata: opts.log_metadata,
       filter_codecs: opts.filter_codecs,
       inbound_tracks: [],
-      outbound_tracks: []
+      outbound_tracks: [],
+      extensions: opts.webrtc_extensions || %{},
```

Comment:
Are you sure this is correct?

Reply:
Ok you are right if nil this should be `[]`.